### DISCUSSION TITLE
Ignite 2020 - miejsce na dyskusję o nowościach

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#77] [Wydarzenia]
Tutaj będzie link do book of news jak wyjdzie! :-)

https://news.microsoft.com/ignite-2020-book-of-news/

**Comment by aryll** [comment]
To ja tu zostawię link do vouchera na egzamin po przejściu ścieżki z Ignite'a ;) 
https://docs.microsoft.com/en-us/learn/certifications/microsoft-ignite-cloud-skills-challenge-2020-free-certification-exam

**Reply by mbiszczanik** [reply]
Coś jeszcze warto ciekawego wyhaczyć, ewentualnie zdobyć jakieś darmowe materiały/pomoce do nauki?
Mówię z perspektywy entry-level :>

**Reply by aryll** [reply]
@bishtou jest w czym wybierać :D chyba kwestia tego w którą stronę chcesz się rozwijać

**Comment by eklime** [comment]
W trakcie konferencji Ignite 2020 zostało ogłoszonych wiele nowości, a wśród nich taka, mówiąca o General Availability WAC w wersji 2009: [Link do newsa](https://techcommunity.microsoft.com/t5/windows-admin-center-blog/windows-admin-center-version-2009-is-now-generally-available/ba-p/1696665)

Dla tych, co tylko nagłówki czytają.
Jest tam taki akapit: "**Windows Admin Center: Now available in the Azure Portal**" 🤨  i link do formularza, w którym można poprosić o dostęp do Private Preview.

A zatem, jeśli ktoś chciałby mieć realny wpływ na wygląd jednej z usług w Azure, to warto się zgłosić - a nuż dostaniesz zaproszenie i dostęp do tej funkcjonalności przed innymi!

Aha...działa ;)
![image](https://user-images.githubusercontent.com/12485979/94407851-4e836480-0174-11eb-8a8e-6013ce802cfa.png)
